libraries/azure_web_app_functions.rb

```diff
@@ -0,0 +1,67 @@
+class AzureWebAppFunctions < AzureGenericResources
+  name 'azure_web_app_functions'
+  desc 'Verifies settings for a collection of Azure PostgreSQL Databases on a PostgreSQL Server'
```

Comment:
Bit of copy-paste here - Note the "PostgreSQL Databases"

Reply:
nice catch actually!